### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jixiangyc1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # testrepo
 test git/github opeartions
+
+add by jixiang1
```
Comment (jixiang1) [inline README.md:4]: I dont' like your modification

**Review by jixiang1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jixiangyc1) [reply README.md:4]: updated

Comment (jixiang1) [reply README.md:4]: good

**Review by jixiang1 — APPROVED** [review]

**Review by jixiang1 — COMMENTED** [review]
Add a test here

**Review by jixiang1 — APPROVED** [review]

**Pull request merged**